src/engine/Operation.h

```diff
@@ -260,13 +262,26 @@ class Operation {
   //! Compute the result of the query-subtree rooted at this element..
   virtual ProtoResult computeResult(bool requestLaziness) = 0;
 
+  // Update the runtime information of this operation according to the given
+  // arguments, considering the possibility that the initial runtime information
+  // was replaced by calling `RuntimeInformation::addLimitOffsetRow`.
```

Comment:
Document (elaborate) what the semantics of `applyToFilter` are, the rest of the arguments are somewhat clear.